documentation/cluster-controller.adoc

```diff
@@ -74,6 +75,8 @@ the related ConfigMap :
  Removing this field means having no metrics exposed.
 * `zookeeper-metrics-config`: a JSON string representing the JMX exporter configuration for exposing metrics from Zookeeper nodes.
  Removing this field means having no metrics exposed.
+* `topic-controller-config`: a JSON string representing the topic controller configuration. See the <<Topic controller>> documentation
+for further details. More info about the topic controller in the related <<Topic Controller>> documentation page.
```

Comment:
Is this not the same forward ref in two successive sentences? Are there really two that differ only by case?

Reply:
Well the first one wants to jump to the following Topic Controller section with JSON config. The second one is referring to the Topic Controller main doc page (topic-controller.adoc).

Reply:
I have updated the reference